.travis.yml

```diff
@@ -38,14 +38,14 @@ matrix:
   include:
     # Default BUILD is "scons".
```

Comment:
We should consider changing the default build to cmake (not in this PR though, let's keep this trivial)